src/ResourceManager/Websites/Commands.Websites/Az.Websites.psd1

```diff
@@ -96,7 +96,10 @@ CmdletsToExport = 'Get-AzAppServicePlan', 'Set-AzAppServicePlan',
     'Reset-AzWebAppPublishingProfile', 'Restart-AzWebApp', 
     'Set-AzWebApp', 'Start-AzWebApp', 'Stop-AzWebApp', 
     'Get-AzWebAppSnapshot', 'Restore-AzWebAppSnapshot',
-    'Get-AzDeletedWebApp', 'Restore-AzDeletedWebApp'
+    'Enter-AzureRmWebAppContainerPSSession', 'Get-AzureRMWebAppContainerContinuousDeploymentUr',
+    'New-AzureRmWebAppContainerPSSession'
```

Comment:
Don't delete `'Get-AzDeletedWebApp'` and `'Restore-AzDeletedWebApp'`. Also, the new cmdlet names would be `Enter-AzWebAppContainerPSSession`, `Get-AzWebAppContainerContinuousDeploymentUrl`, and `New-AzWebAppContainerPSSession` for this file.

Reply:
Done